projects/plutus/test/test_current_sense.c

```diff
@@ -0,0 +1,94 @@
+#include <inttypes.h>
+
+#include "current_sense.h"
+
+#include "gpio.h"
+#include "interrupt.h"
+#include "log.h"
+#include "ltc2484.h"
+#include "soft_timer.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+// Arbitrary number of testing samples
+#define TEST_NUM_SAMPLES 5
+
+// Arbitrary test input voltage
+static int32_t s_test_input_voltage = 0;
+
+static volatile uint8_t s_callback_runs = 0;
+
+static CurrentSenseStorage s_storage = { 0 };
+static LtcAdcSettings s_adc_settings = { 0 };
+static LtcAdcStorage s_adc_storage = { 0 };
+
+static CurrentSenseCalibrationData s_data = { .zero_point = { .voltage = 0, .current = 0 },
```

Comment:
This should be const. Calibration data should not be modified.

Reply:
Done